reporting/impl/src/main/resources/reports/templates/migration-issues.ftl

```diff
@@ -223,37 +222,7 @@
             }
         });
     });
-
-   	    function resizeTables()
-        {
```

Comment:
Won't removing this also result in the different sections (optional vs mandatory) having differing widths?